test/e2e/tests/confirmations/navigation.spec.ts

```diff
@@ -166,11 +165,13 @@ async function queueSignatures(driver: Driver) {
   await driver.waitUntilXWindowHandles(3);
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
   await driver.findElement({ text: 'Reject all' });
+  await driver.waitForSelector(By.xpath("//div[normalize-space(.)='1 of 2']"));
 
   await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
   await driver.clickElement('#signTypedDataV4');
   await driver.waitUntilXWindowHandles(3);
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+  await driver.waitForSelector(By.xpath("//div[normalize-space(.)='1 of 3']"));
```

Comment:
we wait for the navigation to be loaded. Since the inner value have spaces we need to normalize it

![Screenshot from 2024-09-30 09-41-44](https://github.com/user-attachments/assets/958b4fe4-4927-4ec7-b821-bfb3572657a6)

Reply:
@seaona, small nit could it possible to wait for any other stable element in the dialog to load?

Reply:
hi @hjetpoluru I've seen that waiting for the navigation to be loaded ensures that the signatures have been queued. 
We could wait for the signature content, but that's not directly fixing the issue. What were you thinking on?

Reply:
@seaona, I was thinking about waiting for the buttons to load. But after reading your reply, it looks like we need to wait for the navigation to be loaded to ensure the signatures are queued. Maybe we could add a data-testId to the page loader? Otherwise, I will approve this PR as it looks good.

Reply:
Oh perfect @seaona thanks for your patience and explaining this to me.

Reply:
no problem! Thank you for your detailed review @hjetpoluru :bowing_woman: